src/components/EventCalender/Weekly/WeeklyEventCalendar.tsx

```
@@ -0,0 +1,209 @@
+              {expanded === index
+                ? allEventsList
+                : holidayList.length > 0
+                ? allEventsList.slice(0, 1)
+                : allEventsList.slice(0, 2)}
+            </div>
+            {(allEventsList.length > 2 || (windowWidth <= 700 && allEventsList.length > 0)) && (
+              <button
+                className={styles.btn__more}
+                data-testid="more"
+                onClick={() => toggleExpand(index)}
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)